### PR TITLE
Add test_vis_bbox

### PR DESCRIPTION
**Pull request by yuyu2172**
Merge after  #135.

### CODE REVIEW

**Comment by yuyu2172** [comment]
Please merge it if you think it is ready.

File: tests/visualizations_tests/test_vis_bbox.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+class TestVisBbox(unittest.TestCase):
+
+    def test_vis_bbox(self):
+        if optional_modules:
+            img = np.random.randint(
+                0, 255, size=(3, 32, 32)).astype(np.float32)
+            bbox = np.random.uniform(size=(3, 4)).astype(np.float32)
+            ax = vis_bbox(img, bbox, self.label, self.label_names)
+
+            self.assertTrue(isinstance(ax, matplotlib.axes.Axes))
```
Comment (Hakuyume) [inline tests/visualizations_tests/test_vis_bbox.py]: `assertIsInstance` is better

**Comment by yuyu2172** [comment]
Thanks for reviewing. I fixed the code.

**Comment by Hakuyume** [comment]
LGTM

**Pull request merged**